ruby-kafka.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency 'gssapi', '>=1.2.0'
```

Comment:
gssapi requires `ffi`, to map C functions to ruby... I think there's no simple way to rewrite it entirely in ruby, as it's still system dependent (based on system kerberos implementation)